File: hw/otavalo/u2/hw19Inheritance/FarmSystemJavaScript/model/Location.js
class Location {
    constructor(xCoordinate, yCoordinate) {
        this.xCoordinate = xCoordinate;
        this.yCoordinate = yCoordinate;
    }

    toString() {
        return `Location{xCoordinate=${this.xCoordinate}, yCoordinate=${this.yCoordinate}}`;
    }

    getxCoordinate() {
        return this.xCoordinate;
    }

    setxCoordinate(xCoordinate) {
        this.xCoordinate = xCoordinate;
    }

    getyCoordinate() {
        return this.yCoordinate;
    }

    setyCoordinate(yCoordinate) {
        this.yCoordinate = yCoordinate;
    }
}

module.exports = Location;

File: hw/ramos/u2/hw19Inheritance/Java/Farmsystem/src/main/java/ec/edu/espe/farmsystem/model/Location.java
package ec.edu.espe.farmsystem.model;

/**
 *
 * @author Paulo Ramos
 */
public class Location {
    private int xCoordinate;
    private int yCoordinate;

    @Override
    public String toString() {
        return "Location{" + "xCoordinate=" + xCoordinate + ", yCoordinate=" + yCoordinate + '}';
    }

    
    
    public Location(int xCoordinate, int yCoordinate) {
        this.xCoordinate = xCoordinate;
        this.yCoordinate = yCoordinate;
    }



    /**
     * @return the xCoordinate
     */
    public int getxCoordinate() {
        return xCoordinate;
    }

    /**
     * @param xCoordinate the xCoordinate to set
     */
    public void setxCoordinate(int xCoordinate) {
        this.xCoordinate = xCoordinate;
    }

    /**
     * @return the yCoordinate
     */
    public int getyCoordinate() {
        return yCoordinate;
    }

    /**
     * @param yCoordinate the yCoordinate to set
     */
    public void setyCoordinate(int yCoordinate) {
        this.yCoordinate = yCoordinate;
    }
    
    
    
    
}
